memfs.go

```diff
@@ -23,18 +24,40 @@ func NewMemFs() FileSystem {
 	}
 }
 
-func (fs *memFS) Reload(add func(key, name string)) error {
-	return nil
-}
-
-func (fs *memFS) AccessTimes(name string) (rt, wt time.Time, err error) {
+func (fs *memFS) Stat(name string) (FileInfo, error) {
 	fs.mu.RLock()
 	defer fs.mu.RUnlock()
+	f, ok := fs.files[name]
+	if !ok {
+		return FileInfo{}, errors.New("file has not been read")
+	}
+
+	size, err := fs.size(name)
```

Comment:
We already have f from above, so we can just grab int64(len(f.Bytes())) for size right?